home.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="global.css">
+    <title>SnakeGame</title>
+</head>
+
+<body id="home">
+    <div class="game-container">
+        <div class="containerg" id="containerg">
+            <div class="left-box" id="left-box">
+                <h1 class="center">Snake Game</h1>
+
+                <li>Press <b>A/a</b> to increase the size of the snake!</li>
+                <li>Press <b>R/r</b> to reduce the size of the snake!</li>
+
+                <h1 class="h1Speed">Choose your speed</h1>
+                <div class="speed-box">
+                    <li>Press <b>S/s</b> for Slow</li>
+                    <li>Press <b>F/f</b> for Fast</li>
+                    <li>Press <b>T/t</b> for Turbo</li>
+                </div>
+            </div>
+            <button id="container-button" onclick="displayNone()">></button>
+
+        </div>
+        <div style="width: 60%; height: 80%!important; ">
+            <canvas id="game-canvas"
+                style=" background-color:white; border:5px solid #000000; border-radius: 10px; box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.512);">
+            </canvas>
+            <div class="score">
+                <div class="scoreLabel">Score: <span id="score"></span></div>
+                <div class="scoreLabel">High Score: <span id="high-score"></span></div>
+            </div>
+        </div>
+
+    </div>
+
+
+</body>
+<script src="main.js" defer></script>
+<script>
+    function displayNone() {
+        let container = document.getElementById("containerg");
+        let leftBox = document.getElementById("left-box");
+        let button = document.getElementById("container-button");
```

Comment:
For all these  variables, I don't see any reason why they are created via the `let` keyword, it should always be defaulted to `const` unless needed otherwise.